ui/src/main/java/edu/wpi/grip/ui/codegeneration/CppTMethods.java

```diff
@@ -16,6 +16,16 @@ public String name(String name) {
   }
 
   @Override
+  public String getterName(String name) {
+    return "get" + name(name);
```

Comment:
Should be `"get" + WordUtils.capitalize(name(name))`, otherwise you'll get stuff like "getblobsOutput" instead of "getBlobsOutput"